db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  class EmsFolder < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    # Go through all existing EmsFolder records and set :type => Datacenter
+    # if the is_datacenter column is true before deleting it
+    EmsFolder.where(:is_datacenter => true).update_all(:type => "Datacenter")
```

Comment:
In a followup PR, @agrare can you wrap the updates with say_with_time... git grep the db/migrate directory for examples.